# snapshot or record
int8 type
---
int8 status